package.json

```diff
@@ -24,5 +26,10 @@
     "stylelint": "^15.4.0",
     "stylelint-config-recommended": "^11.0.0",
     "stylelint-config-standard": "^32.0.0"
-  }
+  },
+  "type": "module",
```

Comment:
These don't belong here. But I don't think they hurt anything. Still, it would be a good idea to delete lines 29-34 and merge to main again.